tests/test_history.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from banidb.banidb import LRUCache
+
+url = 'https://api.banidb.com/v2'
+target = 'mock.dat'
+cache = LRUCache(target, 25)
+
+
+def test_put():
+    shabad = {
+        "shabadInfo": {
+            "shabadId": 1111,
+            "source": {
+                "unicode": "source-unicode-ਸ੍ਰੀ",
+                "english": "source-english",
+                "pageNo": 111},
+            "writer": {"english": "writer-english"}
+            },
+
+        "verses": [
+                  {
+                      "verseId": 11111,
+                      "verse": {
+                          "unicode": "verse-unicode-ਮਹਲਾ ॥"
+                      },
+                      "larivaar": {
+                          "gurmukhi": "DnwsrImhlw5]",
+                          "unicode": "ਧਨਾਸਰੀਮਹਲਾ੫॥"
+                      },
+                      "translation": {
+                          "en": {
+                              "bdb": "xlate-en-bdb",
+                              "ms": "xlate-en-ms",
+                              "ssk": "xlate-en-ssk"
+                          },
+
+                      },
+                      "transliteration": {
+                          "english": "transliteration-english",
+                          "hindi": "transliteration-हिंदी",
+                          "en": "transliteration-en",
+                          "hi": "transliteration-हिं",
+                          "ipa": "transliteration-ɐ ɑ ɒ ɓ ɔ ɕ ɖ",
+                          "ur": "دھن-transliteration"
+                      },
+
+                  }]
+        }
+    cache.put(1111, shabad)
+    result = cache.get()[1111]
+    assert result == shabad
+    check = cache.check(1111)
+    assert check[0] is True
+
+
+def test_clear():
+    cache.clear()

```

Comment:
Shouldn't you first add something to the cache, clear it and then do the assert?

Reply:
Please keep the two tests separate. From a testing strategy perspective. 1 unit test should test a single piece of functionality.
Why ? because if this test fails, you will not know - without debugging - whether the cache.get failed, or the cache.check failed, or cache.clear failed.

lines 10-48 can be made a `pytest fixture` 
[See pytest fixture documentation](https://docs.pytest.org/en/latest/how-to/fixtures.html)

Reply:
Done👍